### PR TITLE
Pattern template widget dao

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepository.java [hunk 1]
```diff
@@ -253,4 +254,6 @@
 	 * @return list of {@link MostTimeConsumingTestCasesContent}
 	 */
 	List<MostTimeConsumingTestCasesContent> mostTimeConsumingTestCasesStatistics(Filter filter, int limit);
+
+	List<TopPatternTemplatesContent> patternTemplate(Filter filter, Sort sort, String attributeKey, @Nullable String patternName, boolean isLatest, int limit);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepository.java]: documentation

Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepository.java]: comments?!

Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepository.java]: added

Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/WidgetContentRepository.java]: added

**Pull request merged**